src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+
+/* Generic */
+export const cancelText = i18n.translate('cancel', {
+  defaultMessage: 'Cancel',
+});
+
+export const deleteText = i18n.translate('delete', {
+  defaultMessage: 'Delete',
+});
+
+export const titleText = i18n.translate('title', {
+  defaultMessage: 'Title',
+});
+
+export const descriptionText = i18n.translate('description', {
+  defaultMessage: 'Description',
+});
+
+export const usernameText = i18n.translate('username', {
+  defaultMessage: 'Username',
+});
+
+export const passwordText = i18n.translate('password', {
+  defaultMessage: 'Password',
+});
+
+/* Datasource listing page */
+export const dsListingAriaRegion = i18n.translate(
+  'dataSourcesManagement.createDataSourcesLiveRegionAriaLabel',
+  {
+    defaultMessage: 'Data Sources',
+  }
+);
+export const dsListingTitle = i18n.translate('dataSourcesManagement.dataSourcesTable.title', {
+  defaultMessage: 'Data Sources',
+});
+
+export const dsListingDescription = i18n.translate(
+  'dataSourcesManagement.dataSourcesTable.description',
+  {
+    defaultMessage:
+      'Create and manage the data sources that help you retrieve your data from multiple Elasticsearch clusters',
```

Comment:
!!!!!!!!!!

Reply:
will address in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2434, need input from UI/UX